apps/pipelines/views.py

```diff
@@ -94,10 +95,18 @@ def delete(self, request, team_slug: str, pk: int):
 def _pipeline_node_parameter_values(team, llm_providers, llm_provider_models):
     """Returns the possible values for each input type"""
     source_materials = SourceMaterial.objects.filter(team=team).values("id", "topic").all()
-    assistants = OpenAiAssistant.objects.filter(team=team).values("id", "name").all()
+    assistants = OpenAiAssistant.objects.filter(team=team).values("id", "name", "working_version").all()
 
-    def _option(value, label, type_=None):
-        return {"value": value, "label": label} | ({"type": type_} if type_ else {})
+    def _option(value, label, type_=None, edit_url: str | None = None):
+        data = {"value": value, "label": label}
+        data = data | ({"type": type_} if type_ else {})
+        data = data | ({"edit_url": edit_url} if edit_url else {})
+        return data
+
+    def _get_assistant_url(assistant_id, working_version):
+        if working_version is None:
+            return absolute_url(reverse("assistants:edit", args=[team.slug, assistant_id]))
```

Comment:
what about always linking to the working version?

Reply:
https://github.com/dimagi/open-chat-studio/pull/950/commits/07abbdbd835b8526d17b563fa7e090b7ae884d40